lib/c-api/tests/wasmer-c-api-test-runner/src/lib.rs

```diff
@@ -178,12 +178,12 @@ fn test_ok() {
                     &[
                         format!("{}/include/", config.wasmer_dir),
                         format!("{}/wasm-c-api/include/", config.root_dir),
-                        format!("{}", config.root_dir),
+                        config.root_dir.to_string(),
                     ],
                     &mut log,
                     &config.root_dir,
                 )
-                .expect(&format!("failed to fix symlinks: {log}"));
+                .unwrap_or_else(|_| panic!("failed to fix symlinks: {log}"));
```

Comment:
Use expect instead

Reply:
this is a clippy autofix done by clippy --fix --all, it does exactly the same thing, but it doesn't construct a string in the "okay" case. wontfix because it will be reapplied by `clippy --fix` on the next run anyways.